src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -1,3 +1,5 @@
+import { Callout } from "/src/components/callout";
```

Comment:
Can you move this import after the H1? It makes it easier for us to scan and find the page title for rendering.